generators/git/index.js

```diff
@@ -46,17 +46,21 @@ module.exports = class extends Generator {
     );
   }
 
-  writing() {
+  _readPkg() {
```

Comment:
Why do we need to change this?

If we need to call if before each method, then let's change this to be a local variable rather than a function causing a side-effect on `this`.